Doc/whatsnew/3.9.rst

```diff
@@ -352,6 +352,11 @@ Deprecated
 
 Removed
 =======
```

Comment:
Please add an empty line. I'm not sure that it's rendered properly without this newline.

Reply:
Updated!